### PR TITLE
10 Pre-War Clothes 

### PR DESCRIPTION
**Pull request by Bonomedia**
Adds 10 pre-war clothes, doesn't have the extra alt stuff with it

### CODE REVIEW

File: mojave/items/clothing/under/wasteland/under.dm [hunk 1]
```diff
@@ -196,3 +196,71 @@
...
+	can_adjust = TRUE
+	alt_covers_chest = FALSE
+
+/obj/item/clothing/under/ms13/mechanicprewar/mechanicgrey
+	name = "grey mechanic shirt"
+	desc = "A grey mechanic shirt. It smells of oil."
+	icon_state = "mechanicg"
+	inhand_icon_state = "bl_suit"
+	can_adjust = TRUE
+	alt_covers_chest = FALSE
```
Comment (uomo91) [inline mojave/items/clothing/under/wasteland/under.dm:258]: What is `alt_covers_chest` for?

Comment (Bonomedia) [reply mojave/items/clothing/under/wasteland/under.dm:258]: iirc, it checks if the down state covers the chest or not

**Review by uomo91 — APPROVED** [review]

**Review by infraredbaron — APPROVED** [review]
Sprites seems to be in order. If anything it's good to have at least something new.

**Review by uomo91 — CHANGES_REQUESTED** [review]
Please fix the hand template holes, as per our discussion.

**Review by uomo91 — APPROVED** [review]

**Pull request merged**